### ISSUE TITLE
Documentation - Implement Recommendations from ROE Research

### ISSUE THREAD

**Issue by jchasia-adhoc** [department-of-veterans-affairs/va.gov-team#4066]
**Problem:**
-  User research on ROE documentation found VFS teams are building their own link aggregator pages in order to access the relevant documentation from our repository.  We also found people don’t know which documents to trust as the source of truth and don’t have confidence that they are looking at the most up-to-date version.

- A related pain point for VFS teams is they missing having a high-level view of the tasks they will be asked to do through the product development lifecycle showing why and where they fit in to the big picture

**Solution:** 
1.  Update the ReadMe’s for each practice area using this proposed template as applicable:  https://docs.google.com/document/d/10kREoy1dMFS2zUg2z5CJ0OIAIQBJa2JoNootVkjx5yw/edit?usp=sharing

2.  Pin updated ReadMe's to the vfs-platform-support slack channel.

3. Lend a feeling of authority to documentation by adding “Date last updated” and “Team Responsible” as a default part of va.gov-team repo content which also serves to instill a level of responsibility for our teams to maintain the documentation.

**Objectives:**
O1: Any volume of simultaneous customers have a delightful, self-service, end-to-end experience using the Platform to build on VA.gov.

**KPIs:**
- Reduction in average volume of documentation support requests per week
- Increase in customer NPS
- Reduction in volume of “non-human-needed” support requests month over month

**Next Steps:**
- Does the template work for each practice area? 
- As a way to keep the ReadMe as the main message when landing in a practice area folder (it displays below other files in the folder), consider gathering loose internal documents into a folder labeled for internal use (or something like that) to make it clear what VFS members should look at.

**Definition of Done:**
[Practice area](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform) READMEs have been updated in accordance with recommendations: 
- [x] accessibility
- [ ] analytics
- [x] information architecture
- [x] content
- [x] design
- [ ] engineering
  - [ ] operations
  - [ ] frontend tools
  - [ ] backend tools
- [ ] product management

Backlogged READMEs to pick up in the future:
- [ ] call-center
- [ ] quality assurance
- [ ] research
- [ ] security
- [ ] privacy
- [ ] working-with-vsp

**Comment by meganhkelley** [comment]
Per conversation with @gmaciolek , changes to practice area READMEs that fall under VSP's purview must be completed before rollout of docs homepage MVP (#2130)

Editing above issue to include definition of done

**Comment by gmaciolek** [comment]
I'm tracking progress of this work here: https://docs.google.com/document/d/1S357txpjh_6FKNYhpT5P0LHH2VStEXX7baAKJQ8n3XQ/edit?usp=sharing The "Content exploration" section links to drafts for each practice area that I have started work on.

**Comment by Khyatinp18** [comment]
2/16/2022:

Closing this since it is not a priority anymore. Not needed.
